app/views/admin/project_logs/_log_list.html.slim

```diff
@@ -1,25 +1,29 @@
-- if @logs.blank? && @context != 'calendar'
-  = content_tag(:div, t('common.no_records'), class: 'no-records-msg')
+- @context = @context ? @context : 'logs_list'
+- step_modal_context = @context == ('calendar' || 'timeline') ? true : false
```

Comment:
I added this logic to not have to use `if context == `calendar'` in tons of places.

Eventually it would be nice for "calendar" context not to represent both the calendar and timeline table.